docs/api-documenter.json

```diff
@@ -2,5 +2,5 @@
   "outputTarget": "markdown",
   "newlineKind": "lf",
   "uriRoot": "/docs/apis",
-  "onlyPackagesStartingWith": "@fluidframework"
+  "onlyPackagesStartingWith": ["@fluidframework", "@fluid-experimental/property"]
```

Comment:
You'll also need to edit docs/data/packages.yaml to include the relevant packages.